ucfcs/GrowthPlus
# Julio/road map screens2

This PR creates the roadmap screens: 

To test: 
Build and run the app
Click on parent icon -> roadmap icon -> the first roadmap with the bottom bar navigation should appear. 
Switch between the roadmaps and make sure sure the app doesn't crash. 
on click listeners are not set up yet.
